src/config/dynamic-theme-fixes.config

```diff
@@ -620,6 +620,15 @@ INVERT
 
 ================================
 
+bbb*
```

Comment:
What's an example URL?

Reply:
I can't give an example URL. I discovered the issue in the privately hosted BigBlueButton instance of my university. All the rooms have a URL like this: `https://bbb42.fbi.h-da.de` but the number can be different.

Reply:
Ah I see, I think the URL can be changed to bbb*.*.* in order to just specify subdomains that start with `bbb` as something like bbba1234.com could be targetted as well.

Reply:
You can also try a demo at https://demo.bigbluebutton.org, using the `demo[nn].bigbluebutton.org` pattern.